content/src/anytime-action-tasks/operating-carnival-fire-permit.md

```diff
@@ -17,15 +15,15 @@ callToActionLink: https://www.nj.gov/dca/dfs/pdf/fire%20directory_county%20summa
 ---
 
 ## Common Activities That Need a Fire Permit
-* Setting up trailers, tents, stages, and similar structures that people will occupy
-* Setting up a `carnival or amusement ride|carnival-amusement-ride`
-* Setting up food stalls or concession stands that need a fire suppression system, such as fire extinguishers or sprinklers
-* Using an open flame or flame-producing device
-* Storing and handling certain types of flammable or combustible liquids
-* Storing and discharging fireworks
 
-Contact the `Local Enforcing Agency (LEA)|local-enforcing-agency` in the town where your carnival or circus will be to see which fire permits you need and how to apply.
+- Setting up trailers, tents, stages, and similar structures that people will occupy
+- Setting up a `carnival or amusement ride|carnival-amusement-ride`
+- Setting up food stalls or concession stands that need a fire suppression system, such as fire extinguishers or sprinklers
+- Using an open flame or flame-producing device
+- Storing and handling certain types of flammable or combustible liquids
+- Storing and discharging fireworks
 
+Contact the `Local Enforcing Agency (LEA)|local-enforcing-agency` in the town where your carnival or circus will be to see which fire permits you need and how to apply.
 
```

Comment:
dust - a reminder that we should at some point try and unify our editor setting to align with the CMS settings at some point. (not that you should do this in this ticket or anything but a reminder for a broader disscussion)